lib/view/components/home/event_invite_list.dart

```diff
@@ -23,6 +23,7 @@ class _EventInvitesListState extends State<EventInvitesList> {
   Future<void> fetchInvitedEvents() async {
     List<Map<String, dynamic>> invitedEvents = await _eventsService.getInvitedEvents();
 
+    if(mounted) return;
```

Comment:
I think this might need to be if(!mounted) return, right? As far as I understand it, if it is not mounted, then we want to return, because we only want it to resume the function if it is mounted.

Reply:
Yep you are correct, thanks ! I fixed it 👍